src/openfermion/measurements/_rdm_mapping_functions_test.py

```diff
@@ -0,0 +1,187 @@
+
+        phdm_h2_sto3g = os.path.join(
+            THIS_DIRECTORY, 'tests/phdm_H2_sto-3g_singlet_1.4.hdf5')
+        with h5py.File(phdm_h2_sto3g, 'r') as fid:
+            self.phdm_h2_sto3g = fid['phdm'][...]
+
+        tqdm_h2_6_31g = os.path.join(THIS_DIRECTORY,
+                                     'tests/tqdm_H2_6-31g_singlet_0.75.hdf5')
+        with h5py.File(tqdm_h2_6_31g, 'r') as fid:
+            self.tqdm_h2_6_31g = fid['tqdm'][...]
+
+        phdm_h2_6_31g = os.path.join(THIS_DIRECTORY,
+                                     'tests/phdm_H2_6-31g_singlet_0.75.hdf5')
+        with h5py.File(phdm_h2_6_31g, 'r') as fid:
+            self.phdm_h2_6_31g = fid['phdm'][...]
+
+        tqdm_lih_sto3g = os.path.join(
+            THIS_DIRECTORY, 'tests/tqdm_H1-Li1_sto-3g_singlet_1.45.hdf5')
+        with h5py.File(tqdm_lih_sto3g, 'r') as fid:
+            self.tqdm_lih_sto3g = fid['tqdm'][...]
+
+        phdm_lih_sto3g = os.path.join(
+            THIS_DIRECTORY, 'tests/phdm_H1-Li1_sto-3g_singlet_1.45.hdf5')
+        with h5py.File(phdm_lih_sto3g, 'r') as fid:
+            self.phdm_lih_sto3g = fid['phdm'][...]
+
+    def test_krond(self):
+        assert krond(0, 0) == 1
+        assert krond(0, 1) == 0
+        assert krond(1, 0) == 0
+        assert krond(1, 1) == 1
+
+    def test_tpdm_to_opdm(self):
+        # for all files in datadirectory check if this map holds
+        for file in filter(lambda x: x.endswith(".hdf5"),
+                           os.listdir(DATA_DIRECTORY)):
+            molecule = MolecularData(
+                filename=os.path.join(DATA_DIRECTORY, file))
+            if (molecule.fci_one_rdm is not None
+               and molecule.fci_two_rdm is not None):
+                test_opdm = tpdm_to_opdm(molecule.fci_two_rdm,
+                                         molecule.n_electrons)
+                assert numpy.allclose(test_opdm, molecule.fci_one_rdm)
+
+    def test_opdm_to_oqdm(self):
+        for file in filter(lambda x: x.endswith(".hdf5"),
+                           os.listdir(DATA_DIRECTORY)):
+            molecule = MolecularData(
+                filename=os.path.join(DATA_DIRECTORY, file))
+            if molecule.fci_one_rdm is not None:
+                test_oqdm = opdm_to_oqdm(molecule.fci_one_rdm)
+                true_oqdm = numpy.eye(molecule.n_qubits) - molecule.fci_one_rdm
+                assert numpy.allclose(test_oqdm, true_oqdm)
+
+    def test_oqdm_to_opdm(self):
+        for file in filter(lambda x: x.endswith(".hdf5"),
+                           os.listdir(DATA_DIRECTORY)):
+            molecule = MolecularData(
+                filename=os.path.join(DATA_DIRECTORY, file))
+            if molecule.fci_one_rdm is not None:
+                true_oqdm = numpy.eye(molecule.n_qubits) - molecule.fci_one_rdm
+                test_opdm = oqdm_to_opdm(true_oqdm)
+                assert numpy.allclose(test_opdm, molecule.fci_one_rdm)
+
+    def test_tqdm_conversions(self):
```

Comment:
This test has 9 asserts - please break them up / make them smaller. Also, it would be safer not to overwrite the builtin `file` - it looks like you can use `filename` instead everywhere it appears.